components/Card.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable react/no-danger */
+
```

Comment:
Can we dump this? I don't see where we're breaking the `react/no-danger` linting rule in this file.

```suggestion
```